resources/sdk/purecloudjavascript/templates/README.mustache

```diff
@@ -160,6 +160,71 @@ client.refreshCodeAuthorizationGrant(clientId,clientSecret,refreshToken)
   });
 ```
 
+**Node.js** [PKCE Grant](https://developer.genesys.cloud/authorization/platform-auth/use-pkce)
+
+```javascript
+const client = {{moduleName}}.ApiClient.instance;
+client.authorizePKCEGrant(clientId,codeVerifier,authCode,redirectUri)
+  .then(() => {
+    // Do authenticated things
+  })
+  .catch((err) => {
+    // Handle failure response
+    console.log(err);
+  });
+```
+
+The SDK provides methods to generate a PKCE Code Verifier and to compute PKCE Code Challenge.
+
+```javascript
```

Comment:
Maybe put this code block in the code block above (line 165) just so people know it's related.